blinds/blinds_schedule.py

```diff
@@ -331,44 +331,36 @@ def hasConflict( timeBlockList ):
         return False
 
     '''
-    Returns a JSON representation of a valid BlindsSchedule object.
-    Provides additional keyword arguments for pretty printing and sorting the json output's keys. 
+    Returns a JSON-like dictionary representation of the BllindsSchedule object.
```

Comment:
Typo: `BllindsSchedule` should be `BlindsSchedule`

Reply:
Fixed